features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/AppNotRespondingDisabledScenario.kt

```diff
@@ -11,12 +12,14 @@ internal class AppNotRespondingDisabledScenario(config: Configuration,
                                   context: Context) : Scenario(config, context) {
     init {
         config.setAutoCaptureSessions(false)
-        config.detectAnrs = false
     }
 
     override fun run() {
         super.run()
-        Thread.sleep(6000)
+        val main = Handler(Looper.getMainLooper())
+        main.postDelayed({
+            Thread.sleep(60000) // FOREVER (test suite doesn't actually wait this long)
+        }, 1) // A moment of delay so there is something to 'tap' onscreen
```

Comment:
I tried this with:
```kotlin
override fun onActivityStarted(activity: Activity) {
  Thread.sleep(6000)
}
```

But it didn't actually seem to block the main thread, or otherwise trigger an ANR.

Reply:
Happy to leave this as-is for now.